apps/website/sw.tsup.config.ts

```diff
@@ -20,7 +20,10 @@ export default defineConfig((options) => ({
     options.define = {
       ...Object.fromEntries(
         Object.entries(env)
-          .filter(([, value]) => value !== undefined)
+          .filter(
+            ([, value]) =>
+              value !== undefined && value.startsWith("NEXT_PUBLIC_")
```

Comment:
added this here because this was originally client only env, wasnt entirely sure how this code runs so added in additional check to make sure client only env is accessed